packages/askar/src/wallet/AskarBaseWallet.ts

```diff
@@ -431,6 +431,27 @@ export abstract class AskarBaseWallet implements Wallet {
     }
   }
 
+  public getRandomValues(buffer: Uint8Array): Uint8Array {
```

Comment:
I think it's more Credo-like to pass in a value length and get back an UInt8array instead of passing in a uint8array with length